src/main/java/com/ecommerce/controller/AdminController.java

```diff
@@ -0,0 +1,47 @@
+package com.ecommerce.controller;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import com.ecommerce.dto.OrderDTO;
+import com.ecommerce.service.AdminService;
+import com.ecommerce.service.impl.AdminServiceImpl;
+
+import lombok.RequiredArgsConstructor;
+
+@Controller
+@RequestMapping("/admin")
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class AdminController {
+	
+	@Autowired
+	private AdminService adminService;
+	
+	@GetMapping("/order")
```

Comment:
```suggestion
	@GetMapping("/orders")
```
ri hì, mà đưa orders này lên chỗ line 19 đi em để đỡ lặp ở các action bên dưới trong controller này